fimsauthz-api/fims-authz.yaml

```diff
@@ -435,3 +480,41 @@ components:
         error_description:
           type: string
           example: "Human readable description of the error."
+    JsonWebKeySet:
+      type: "object"
+      properties:
+        keys:
+          type: "array"
+          items:
+            $ref: "#/components/schemas/JsonWebKey"
+    JsonWebKey:
+      type: "object"
+      description: >-
+        This schema defines what a client may expect to receive when requesting  a JWK/JWKS. 
+
+        NOTE: Not all fields defined in this schema may be returned when in operational use.
+      properties:
+        kty:
+          type: "string"
+        use:
```

Comment:
Recommend enum:

https://tools.ietf.org/html/rfc7517#section-4.2

Values defined by this specification are:

   o  "sig" (signature)
   o  "enc" (encryption)